cmd/juju/caas/add_test.go

```diff
@@ -1166,7 +1118,7 @@ func (s *addCAASSuite) TestFoundStorageProvisionerViaAnnationForMAASWIthoutStora
 		c.Assert(err, jc.ErrorIsNil)
 		result := strings.Trim(cmdtesting.Stdout(ctx), "\n")
 		result = strings.Replace(result, "\n", " ", -1)
-		c.Assert(result, gc.Equals, `k8s substrate "myk8s" added as cloud "myk8s" with storage provisioned by the existing "mystorage" storage class. You can now bootstrap to this cloud by running 'juju bootstrap myk8s'.`)
+		c.Assert(result, gc.Equals, `k8s substrate "myk8s" added as cloud "myk8s". You can now bootstrap to this cloud by running 'juju bootstrap myk8s'.`)
```

Comment:
Maybe we retain the storage message, but just reword it, eg

k8 substrate "foo" added as cloud "bar" with storage provisioned by the storage class "blah".

Or maybe the user doesn't care so long as storage is available, especially if it comes from the cluster default storage class. So on second thought, seems ok I think.